### PR TITLE
update with nepalgunj information

### PR DESCRIPTION
**Pull request by nawarajshah**
## What kind of change does this PR introduce?

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added a new city
- [ ] Other(s)

## Why was this change needed?

There is no information about Nepalgunj so I added it here.

### CODE REVIEW

File: data/borderlines.json [hunk 1]
```diff
@@ -1,5 +1,51 @@
 {
   "cities": [
+    {
+      "name": "Nepalgunj",
```
Comment (susanhsrestha) [inline data/borderlines.json:4]: The city names should be arranged alphabetically.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/borderlines.json (hunk 1 above)
Comment (nawarajshah) [reply data/borderlines.json]: done

File: data/borderlines.json [hunk 2]
```diff
@@ -351,6 +305,52 @@
           "latitude":85.3273
         }
       ]
+  },
```
Comment (susanhsrestha) [inline data/borderlines.json:308]: Format the code using prettier.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]
Review the suggestions.

File: data/borderlines.json (hunk 2 above)
Comment (nawarajshah) [reply data/borderlines.json]: updated with suggestion, added indention in front

**Review by susanhsrestha — APPROVED** [review]
Looks good to me!

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please properly format the code with prettier and then make a commit.

**Comment by nawarajshah** [comment]
> Please properly format the code with prettier and then make a commit.

Can you tell me what I need to improve in the file with a specific reason instead of vage comment like use "prettier"

**Comment by susanhsrestha** [comment]
> > Please properly format the code with prettier and then make a commit.
> 
> Can you tell me what I need to improve in the file with a specific reason instead of vage comment like use "prettier"

Format all of the code and push the modifications.

**Comment by thenishantsapkota** [comment]
> > Please properly format the code with prettier and then make a commit.
> 
> Can you tell me what I need to improve in the file with a specific reason instead of vage comment like use "prettier"

Prettier isn't a vauge word, it's a code formatter.

**Comment by nawarajshah** [comment]
> > > Please properly format the code with prettier and then make a commit.
> > 
> > 
> > Can you tell me what I need to improve in the file with a specific reason instead of vage comment like use "prettier"
> 
> Prettier isn't a vague word, it's a code formatter.

i know what prettier, I have install and use it for many years. i don't see my updated data format is different then other who updated the city info. If I don't see the difference then how can I update the code format of mine.

**Comment by thenishantsapkota** [comment]
> > > > Please properly format the code with prettier and then make a commit.
> > > 
> > > 
> > > Can you tell me what I need to improve in the file with a specific reason instead of vage comment like use "prettier"
> > 
> > 
> > Prettier isn't a vague word, it's a code formatter.
> 
> i know what prettier is, I have install and use it for many years. i don't see my updated data format is different then other who updated the city info. If I don't see the difference then how and what can I update the code format of mine.

I've tested the code and saw that the code isn't properly formatted using Prettier. I myself have tried running prettier on your code and it updates the code. Try running the code formatter on `cities.json` once and if nothing changes, I'll merge the PR.

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @nawarajshah

**Pull request merged**